scripts/firmware.py

```diff
@@ -0,0 +1,2097 @@
+import os
+import re
+import subprocess
+import argparse
+import shutil
+import tempfile
+import fnmatch
+import platform
+import time
+
+from enum import Enum  # type: ignore

```

Comment:
```suggestion
from enum import Enum
```
These are bad and shouldn't be used. Minimum should be `pyright: ignore[...]` as we do in scanner.py
Why is this necessary?

Reply:
Im not actually sure what caused this? Its removed.